lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb

```diff
@@ -477,6 +477,9 @@ def add_ecm_hs08_ccashp_vrf(
     standard:,
     air_sys_eqpt_type: 'ccashp')
 
+    # Create one hot-water loop for hot-water baseboards if primary heating fuel is gas
+    hw_loop = add_hotwater_loop(model: model, heating_fuel: heating_fuel)
```

Comment:
here you can initialize the hw_loop = nil and then call the method 'add_hot_water_loop' only when the fuel is NG